src/oracles/QuickswapV3Oracle.sol

```diff
@@ -61,6 +61,7 @@ contract QuickswapV3Oracle is INFTOracle, Ownable {
         returns (
             bool deviationSafety,
             uint256 positionAmount,
+            uint24 width,
             address pool
```

Comment:
Won't be better to return (uint256, bool, uint24, address) for better packing?
wdyt

Reply:
Efficient packing only applies for storage, memory items always occupy a multiple of 32 bytes - https://docs.soliditylang.org/en/v0.8.18/internals/layout_in_memory.html